subsys/jwt/jwt_legacy_ecdsa.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2024 BayLibre SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <zephyr/types.h>
+#include <errno.h>
+
+#include <zephyr/data/jwt.h>
+#include <zephyr/data/json.h>
+#include <zephyr/random/random.h>
+
+#include <tinycrypt/ctr_prng.h>
+#include <tinycrypt/sha256.h>
+#include <tinycrypt/ecc_dsa.h>
+#include <tinycrypt/constants.h>
+
+#include "jwt.h"
+
+static TCCtrPrng_t prng_state;
+static bool prng_init;
+
+static const char personalize[] = "zephyr:drivers/jwt/jwt.c";
+
+static int setup_prng(void)
+{
+	if (prng_init) {
+		return 0;
+	}
+	prng_init = true;
+
+	uint8_t entropy[TC_AES_KEY_SIZE + TC_AES_BLOCK_SIZE];
+
+	sys_rand_get(entropy, sizeof(entropy));
+
+	int res = tc_ctr_prng_init(&prng_state, (const uint8_t *) &entropy, sizeof(entropy),
+				   personalize, sizeof(personalize));
```

Comment:
I'd suggest fixing this, not using a space after a cast is near universal in C style conventions.